src/Microsoft.Diagnostics.Tools.RuntimeClient/Eventing/EventPipeClient.cs

```diff
@@ -0,0 +1,171 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.IO.Pipes;
+using System.Linq;
+using System.Net.Sockets;
+using System.Runtime.InteropServices;
+using System.Security.Principal;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Diagnostics.Tools.RuntimeClient.Eventing
+{
+    public static class EventPipeClient
+    {
+        // TODO: Make async.
+        // TODO: Required an interface (e.g. "ISerializableToByteArray").
+        // TODO: Maybe require DiagnosticMessageType separate.
```

Comment:
I didn't see any additional value added by doing this. I'm not necessarily opposed if there is some benefit I missed, but otherwise I wouldn't make this change.